### DISCUSSION TITLE
BTP Solidity resources

### DISCUSSION THREAD

**Discussion by flowersinthesand** [icon-project/btp#12] [General]
Fixed

- [BMC interface](https://docs.google.com/document/d/1JygezJqliHhXbsVYpAZMkm2RUOTRs0n1d-oQVzXrZm8/edit)
- [BSH interface](https://docs.google.com/document/d/1zDU5XSyY6di-AbD8BBGzbSLEu5PM_sTvINEpvgx8mII/edit)
- [NativeCoinBSH interface](https://docs.google.com/document/d/19hdTuufLzSgyq6A2meJ3nkrgbtNa7Wy9sxOFijjjyJs/edit)

In review

- [Upgradability proposal](https://docs.google.com/document/d/1Mupm9I8PArrHzZ1oR-n4JVyl_tcyJ66YUCMtBuSOipc/edit)
- [BTP 1.1 Test Scenario](https://docs.google.com/document/d/1Iwb0z7rdUagLP1Q_3P8EaT9ats_mmb7LCbnsRpjDSWk/edit#heading=h.xwg3hpzf5zs0)

**Comment by unamed12** [comment]
Thank u!
